server/src/main/java/com/paypal/selion/utils/process/AbstractProcessHandler.java

```diff
@@ -72,7 +72,7 @@
                     tProcess = new ProcessInfo(eachProcessData[1], eachProcessData[2]);
                     break;
                 }
-                if (matches(tProcess.getProcessName())) {
+                if (matches(tProcess.getProcessName().toLowerCase())) {
```

Comment:
Since we are already formatting the `processName` in https://github.com/paypal/SeLion/blob/develop/server/src/main/java/com/paypal/selion/pojos/ProcessInfo.java, can we just do this there?